optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/common/TimeoutRestCaller.java

```diff
@@ -0,0 +1,50 @@
+package org.optaplanner.openshift.employeerostering.gwtui.client.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gwt.user.client.Timer;
+
+public class TimeoutRestCaller {
```

Comment:
@mbarkley Do we have anything like this in errai out of the box?

Reply:
No, I don't think so.